### PR TITLE
renderer: delete collapse related funcs and property

### PR DESCRIPTION
**Pull request by pftom**
* Delete custom.d.ts as we don't need svg, so we don't need declare this file
* Delete StepDiff.tsx state as we don't need it any more

### CODE REVIEW

File: src/components/StepDiff.tsx [hunk 1]
```diff
@@ -21,10 +18,6 @@ interface StepDiffProps {
   renderExplain: RenderExplainFunc;
 }
 
-interface StepDiffState {
-  collapseObj: CollapseObj;
-}
-
 interface CollapseObj {
```
Comment (mRcfps) [inline src/components/StepDiff.tsx]: It seems this `CollapseObj` should be deleted.

File: src/components/StepDiff.tsx [hunk 2]
```diff
@@ -97,30 +63,6 @@ export default class StepDiff extends React.PureComponent<
...
-      ...collapseObj,
-      [fileName]: !collapseObj[fileName],
-    };
-
-    this.setState({
-      collapseObj: newCollapseObj,
-    });
-  };
-
   getCollapseObj = (arr: ChangedFile[]): CollapseObj => {
```
Comment (mRcfps) [inline src/components/StepDiff.tsx]: This `getCollapseObj` is also redundant.

**Comment by mRcfps** [comment]
BTW I found those files not needed either, please delete them.

- **src/components/img/*.svg**
- `collapse` property of `ChangedFile` interface within **src/types/index.ts**

You can search globally for `collapse` to make sure it's thoroughly removed.

File: src/components/StepDiff.tsx (hunk 2 above)
Comment (pftom) [reply src/components/StepDiff.tsx]: We need maintain an array of need hide files, uh !

**Comment by pftom** [comment]
@mRcfps  Very appreciate your great work !

**Pull request merged**